proto/queryservice.proto

```diff
@@ -61,6 +61,12 @@ service Query {
   // BeginExecuteBatch executes a begin and a list of queries.
   rpc BeginExecuteBatch(query.BeginExecuteBatchRequest) returns (query.BeginExecuteBatchResponse) {};
 
+  // MessageStream streams messages from a message table.
+  rpc MessageStream(query.MessageStreamRequest) returns (stream query.MessageStreamResponse) {};
+
+  // MessageAck acks messages for a table.
+  rpc MessageAck(query.MessageAckRequest) returns (query.MessageAckResponse) {};
```

Comment:
I hate to say this, but this can hack multiple messages, should this be 'MessagesAck' with an S?